src/components/chart-wrapper/index.js

```diff
@@ -32,7 +32,7 @@ export const withWrapper = Chart => {
   const ChartWrapper = ({ title, ...chartProps }) => (
     <Wrapper>
       {title.length !==0 && <Title>{title}</Title>}
-      <ChartContainer title={title}>
+      <ChartContainer chartTitle={title}>
```

Comment:
the rename, is it necessary? doesn't really matter tho as this is an "internal wiring" not exposed to users of this component directly

Reply:
yes it is! Because 'title' in an html element is an html attribute, that's why the pointer cursor displayed it when we hovered over the chart. So we have to rename into something that html hasn't set aside already for a pointer-event. https://www.w3schools.com/tags/att_global_title.asp